### PR TITLE
[stable/fluent-bit] Fix indentation of extra params

### PR DESCRIPTION
**Pull request by naseemkullah**
#### What this PR does / why we need it:

Using the extra outputs value, configmap would be produced like this:

```
    [OUTPUT]
        Name  es
        Match *
        Host  elasticsearch-client
        Port  9200
        Logstash_Format On
        Retry_Limit False
        Type  flb_type
        Time_Key @timestamp
        Logstash_Prefix kubernetes_cluster



    Replace_Dots On
```

Where the added `Replace_Dots On` is not indented properly, this PR fixes that.

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/fluent-bit/templates/config.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ data:
         DB               /tail-db/tail-containers-state.db
         DB.Sync          Normal
 {{- end }}
-{{ .Values.extraInputs | indent 4 }}
+        {{ .Values.extraInputs }}
```
Comment (unguiculus) [inline stable/fluent-bit/templates/config.yaml]: This only works for a single line. I think the correct fix would be this:

`{{ .Values.extraInputs | indent 8 }}`

**Review by unguiculus — CHANGES_REQUESTED** [review]
cc @kfox1111 @edsiper

File: stable/fluent-bit/templates/config.yaml (hunk 1 above)
Comment (naseemkullah) [reply stable/fluent-bit/templates/config.yaml]: OK adjusted as per your recommendation!

**Comment by kfox1111** [comment]
yeah, this looks good.

I'm thinking we probably need a setting to disable configmap entirely and let the user load their own. As we add more and more options, it is probably easier at some point for the user just to manage the whole config file directly.

**Comment by naseemkullah** [comment]
@kfox1111 
I believe that is implemented already with the `existingConfigMap: ""` value

**Comment by kfox1111** [comment]
Your right. Sorry. Guess its time to go home for the day.... :)

**Comment by naseemkullah** [comment]
@unguiculus it's ready!

**Review by unguiculus — APPROVED** [review]

**Comment by Towmeykaw** [comment]
This change seems to have broken the purpose of the extra settings. I created it to be able to add entire sections and not just parameters in existing sections. For example when needing multiple elasticsearch outputs

**Comment by naseemkullah** [comment]
@Towmeykaw My apologies!
 I actually thought it was to add parameters to existing sections, I will open a PR shortly to revert this, as well as to include a way to add parameters to existing sections!

**Comment by Towmeykaw** [comment]
Sounds great 😊👍🏼

**Comment by naseemkullah** [comment]
https://github.com/helm/charts/pull/9008 created @Towmeykaw 
I'll forgo the extra params per section for now as https://github.com/helm/charts/pull/8659/ will solve my problem

**Comment by max-lobur** [comment]
https://github.com/helm/charts/pull/9008#issuecomment-439433178

**Pull request merged**